bitnami/logstash/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
In other charts we also allow setting the service account name and we also allow setting an existing service account. Could you check the example in other charts like grafana operator? https://github.com/bitnami/charts/blob/master/bitnami/grafana-operator/templates/serviceaccount.yaml